.github/workflows/build-and-deploy.yml

```diff
@@ -270,112 +270,112 @@ jobs:
         SLACK_MESSAGE: ':alert: <!channel> Publish Teacher Training Test failure :sadparrot:'
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
-  deploy-v2-review-app:
-    name: Deployment To Review v2
-    concurrency: deploy_v2_review_${{ github.event.pull_request.number }}
-    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'deploy_v2') }}
+  deploy-review-app:
+    name: Deployment To Review
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'deploy') }}
```

Comment:
Can we change the label from 'deploy' to 'Review App'?

Reply:
@saliceti what do you think? It will be more explicit about what it does.

Reply:
We use 'deploy' in all our repo's (hopefully) and also in the new service templates, so the preference is to keep it the same.